OG_Format.adoc

```diff
@@ -268,7 +268,7 @@ OG1.0 requirements cover positioning variables and geolocation of any scientific
 * data type: double
 * dimension: N_MEASUREMENTS |
 
-* long_name = “longitude of each measurement and GPS location”;
+* long_name = “longitude east”;
```

Comment:
can you add capital 'L' to match vocab description

Reply:
Done